cleanlab/object_detection/rank.py

```diff
@@ -107,7 +119,11 @@ def get_label_quality_scores(
         method=method,
         threshold=probability_threshold,
     )
-
+    if aggregation_weights is not None:
+        global CUSTOM_SCORE_WEIGHT_OVERLOOKED, CUSTOM_SCORE_WEIGHT_SWAP, CUSTOM_SCORE_WEIGHT_BADLOC
```

Comment:
why global

Reply:
where is the check that the aggregation_weights are properly formatted that I requested?

Reply:
Fixed in commit: https://github.com/cleanlab/cleanlab/pull/676/commits/289827ed8a5caf0c7dba4673cb63a894f75407a7